_episodes_rmd/05-day4.Rmd

```diff
@@ -1,135 +1,173 @@
+Notice the default colors and how they are associated with the data values. Do you think this color scale is a good choice for the data?
+Can you easily read the different color values on the plot, and connect them to the data value? Is the color range intuitive to understand? Could this be symbolized better?
+
+It is a common convention in map symbology to use darker colors for higher data values. We can do this in ggplot2 by reversing the color range. 
+Add the line `scale_fill_continuous(trans = 'reverse') +`
+
+> ## Solution
+> ```{r, eval=TRUE}
+> #Plot a shapefile, reverse the colors
+> ggplot() +
+>  geom_sf(data = wi_precip, aes(fill = PrecipInch), size = 0.1) +
+>  scale_fill_continuous(trans = 'reverse') +   ## reverses color scale     
+>  ggtitle("Wisconsin Annual Precipitation Average 1981 - 2010, In Inches") +
+>  coord_sf()
 ```
+{: .solution}
+
+## 7) Plot multiple vector shapefiles
+You can use ggplot2 to plot mutiple vector shapefiles layered in a single plot. For example, adding a layer of the county outlines over the plot of precipitation with a second use of the `geom_sf()` command. One design option to try is to make choices about the color and thickness of the county lines.
+
+
+> ## Solution
+> ```{r, eval=TRUE}
+> ggplot() +
+>   geom_sf(data = wi_precip, aes(fill = PrecipInch), size = 0.1) +
+>   geom_sf(data = wi_county, fill = NA, color = gray(.5)) +  
+>   scale_fill_continuous(trans = 'reverse') +        ## reverses color scale
+>   ggtitle("Wisconsin Annual Precipitation Average 1981 - 2010, In Inches") +
+>   coord_sf()
+> ```
+{: .solution}
+
+
+## Bonus: Try different color palette
+R offers a number of color packages that allow you to choose from a variety of color ranges. One example of a color palette is  `colorspace` 
+
+The `hcl_palettes` function will print the colorspace palette to the screen
 
-## Including Plots
 
-You can also embed plots, for example:
+ ```{r, eval=TRUE}
+ # install.packages("colorspace")
+ library(colorspace)
+ hcl_palettes(plot = TRUE)
+ ```
 
-```{r pressure, echo=FALSE}
-plot(pressure)
+You can assign colors from a palette to the unique values in a range, as in our precipitation data. First, determine the number of unique values.
+
+
+ ```{r, eval=TRUE}
+ wi_precip_uniq <- unique(wi_precip$Inches)
+ length(wi_precip_uniq)
+ ```
+
+
+In the case of our precipitation data, there are 11 unique values. This code example selects the palette "Purples 3" and reverses the order of the colors.
+
+```{r, eval=TRUE}
+colors <- sequential_hcl(11, palette = "Purples 3")  #11 categories
+colors <- rev(colors)   #reverses order
 ```
 
-Note that the `echo = FALSE` parameter was added to the code chunk to prevent printing of the R code that generated the plot.
+Commands to plot the map are similiar to previous examples, with the addition of the `cut_interval()` function which creates groups with equal ranges of values.
```

Comment:
The cut_intervals() function is a little confusing to me. I think it would be helpful to add an explanation here about why this is necessary, and a longer description about what the function does.